spec/requests/service_templates_spec.rb

```diff
@@ -154,6 +162,17 @@
       expect(st.reload.name).to eq("Updated Template Name")
     end
 
+    it "supports picture update with base 64 decoded content" do
+      api_basic_authorize collection_action_identifier(:service_templates, :edit)
+
+      st = FactoryBot.create(:service_template, :name => "st1")
```

Comment:
Sorry, just noticed this... Please don't set `:name` since the factory already sequences that.